packages/pwa/app/commerce-api/auth.js

```diff
@@ -41,24 +45,20 @@ class Auth {
         this._onClient = typeof window !== 'undefined'
         this._pendingAuth = undefined
         this._customerId = undefined
-
-        this._oid = this._onClient ? window.localStorage.getItem(oidStorageKey) : undefined
+        this._storage = new LocalStorage()
```

Comment:
I am just leaving a comment from the Slack chat. 
It would be nice to add a configuration value setting the App browser storage.
I believe we can define that setting here:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/095d3a8e39913de41d165c1d67c27448f365746f/packages/pwa/config/default.js#L9